src/Workspaces/CoreTestUtilities/Roslyn.Services.UnitTests.Utilities.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.UnitTests</RootNamespace>
-    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
+    <TargetFrameworks>net5.0;net472</TargetFrameworks>
```

Comment:
IDE will likely be on 3.1 until Net 6 in available though.

Reply:
Okay so you all want to stay on `netcoreapp3.1` for your testing?

Reply:
Note: at the moment you are not testing on `netcoreapp3.1`. You are testing on `net5.0`. The current setup for our branch just means you compile fro `netcoreapp3.1`

Reply:
Okay backed this out and kept the `netcoreapp3.1` portion in addition to `net5.0`